### PR TITLE
x11-misc/autocutsel: add version 0.10.0

### PR DESCRIPTION
**Pull request by atalax**

### CODE REVIEW

File: x11-misc/autocutsel/autocutsel-0.10.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# $Id$
+
+DESCRIPTION="Synchronise the two copy/paste buffers mainly used by X applications"
+HOMEPAGE="http://www.nongnu.org/autocutsel/"
+SRC_URI="https://github.com/sigmike/${PN}/releases/download/${PV}/${P}.tar.gz"
+IUSE=""
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="amd64 ppc sparc x86"
```
Comment (hasufell) [inline x11-misc/autocutsel/autocutsel-0.10.0.ebuild]: We don't add stable keywords to new versions. These should be
```
KEYWORDS="~amd64 ~ppc ~sparc ~x86"
```

File: x11-misc/autocutsel/autocutsel-0.10.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
```
Comment (hasufell) [inline x11-misc/autocutsel/autocutsel-0.10.0.ebuild]: This is an opportunity to add a modern EAPI, e.g.
```
EAPI=5
```

File: x11-misc/autocutsel/autocutsel-0.10.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+RDEPEND="x11-libs/libX11
+	x11-libs/libXmu
+	x11-libs/libXt
+	x11-libs/libXaw
+	x11-libs/libXext"
+DEPEND="${RDEPEND}
+	x11-proto/xproto"
+
+src_install() {
```
Comment (hasufell) [inline x11-misc/autocutsel/autocutsel-0.10.0.ebuild]: in case EAPI=5 is added above, I think this whole phase definition is obsolete and already implicit

**Comment by atalax** [comment]
Updated to EAPI=5 and fixed the keywords

**Comment by mgorny** [comment]
@gentoo/desktop-misc

**Comment by monsieurp** [comment]
Is someone looking into merging this PR? I will do it otherwise.

**Comment by mgorny** [comment]
Looks like @jer-gentoo did the version bump in 86c731994aa9784f7f650a085232c8b9 with some more changes.

**Pull request closed without merging**